backend/spec/controllers/spree/admin/users_controller_spec.rb

```diff
@@ -106,6 +106,13 @@
       ))
       spree_put :update, { :id => mock_user.id, :user => { :bill_address_attributes => { :city => "New York" } } }
     end
+
+    it "allows updating without password resetting" do
+      expect(mock_user).to receive(:update_attributes).with(hash_not_including(
+        "password" => "", "password_confirmation" => ""
+      ))
+      spree_put :update, { :id => mock_user.id, :user => { :password => '', :password_confirmation => '', :email => 'spree@example.com' }}
```

Comment:
Line is too long. [138/100]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside } missing.